website/source/docs/agent/options.html.md

```diff
@@ -821,15 +821,14 @@ default will automatically work with some tooling.
 
     The following sub-keys are available:
 
-    * <a name="bootstrap"></a><a href="#config_entries_bootstrap">`bootstrap`</a>
-        This object allows configuring centralized config entries to be bootstrapped
-        by the leader. These entries will be reloaded during an agent config reload.
+    * <a name="config_entries_bootstrap"></a><a href="#config_entries_bootstrap">`bootstrap`</a>
+        This is a list of inlined config entries to insert into Raft when the Consul server
```

Comment:
Should we say "raft" or "the state store" here? Not sure if we generally refer to it one way or the other for user clarity.